assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -131,6 +132,8 @@ function MediaPane(props) {
     insertElement: state.actions.insertElement,
   }));
 
+  const [hasRestrictAccess, setHasRestrictAccess] = useState(false);
```

Comment:
```suggestion
  const [isPermissionDialogOpen, setisPermissionDialogOpen] = useState(false);
```

Reply:
Done. And I kept `setIsPermissionDialogOpen` instead of `setisPermissionDialogOpen` for camel case naming.